### ISSUE TITLE
Mergify configuration is invalid

### ISSUE THREAD

**Issue by kt86** [matsim-org/matsim-libs#1837]
Hi,

I just tried to use our automerge label. But it will not merge, because of the "Mergyfi configuration is invalid" error:
![grafik](https://user-images.githubusercontent.com/12124568/149315997-32961a2a-a065-4977-85d3-a34d06c4f509.png)

Going to the details, says:

"The Mergify configuration is invalid

extra keys not allowed @ pull_request_rules → item 0 → actions → merge → strict"
(https://github.com/matsim-org/matsim-libs/pull/1836/checks?check_run_id=4800663271)

Can anyone help/fix it, or are we now using another automerge setting?

Thanks,
Kai MT

**Comment by rakow** [comment]
There should be an automerge button natively available in GitHub nowadays. Not sure if this automerge label is still needed at all.

**Comment by jfbischoff** [comment]
Yes, just use the github-included automerge functionality. That said, we should remove mergify.

**Comment by jfbischoff** [comment]
This needs to be done by a repo-owner:
https://github.com/apps/mergify --> Configure.
